etcdctl/ctlv3/command/global.go

```diff
@@ -79,6 +83,14 @@ func initDisplayFromCmd(cmd *cobra.Command) {
 func mustClientFromCmd(cmd *cobra.Command) *clientv3.Client {
 	flags.SetPflagsFromEnv("ETCDCTL", cmd.InheritedFlags())
 
+	debug, derr := cmd.Flags().GetBool("debug")
+	if derr != nil {
+		ExitWithError(ExitError, derr)
```

Comment:
huh? it's 1 since it follows ExitSuccess and ExitSuccess = iota-- see https://github.com/coreos/etcd/blob/master/etcdctl/ctlv3/command/error.go#L27